modules/flowable-cmmn-rest/src/main/java/org/flowable/cmmn/rest/service/api/runtime/task/TaskCollectionResource.java

```diff
@@ -310,6 +310,17 @@ public DataResponse<TaskResponse> getTasks(@ApiParam(hidden = true) @RequestPara
         if (requestParams.containsKey("category")) {
             request.setCategory(requestParams.get("category"));
         }
+        if(requestParams.containsKey("withoutCategory") && Boolean.valueOf(requestParams.get("withoutCategory"))){
```

Comment:
Please match the project formatting and include a space in `if(` and `){`; thanks.   There are multiple occurrences of this in multiple files in this PR.

Reply:
should be fixed